modules/platen/layouts/partials/platen/markup/subscript/image.html

```diff
@@ -18,33 +18,39 @@
 
 {{- $hasPrefix := false -}}
 {{- $alt       := $text -}}
-{{- $class     := ""    -}}
+{{- $class     := slice -}}
```

Comment:
I don't think this needs to be a slice - everywhere you munge it, you use `printf` to do so. I don't think it hurts either, though.